### ISSUE TITLE
Time to full charge is always current dateTime if Tesla sends time_to_full_charge = 0.0

### ISSUE THREAD

**Issue by JumiranK** [Teslemetry/hacs-teslemetry#24]
### Checklist

- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of any [previous issues](https://github.com/teslemetry/hacs-teslemetry/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Everytime the integration pulls new data the sensor.<car-name>_time_to_full_charge get the current date time.
Tesla is sending "time_to_full_charge": 0.0. It looks like the availabilty logic is not working.

### Reproduction steps

1. Check via postman that the field charge_state.time_to_full_charge is 0.0
2. Reload the integration
3. Notice that sensor.<car-name>_time_to_full_charge gets the current date time.

**Comment by Bre77** [comment]
Yes, this is a regression, it will be fixed in the next release.
